app/components/support_interface/qualification_documentation_component.html.erb

```diff
@@ -0,0 +1,14 @@
+<details class="govuk-details" data-module="govuk-details">
```

Comment:
This is just the `govuk-details` component and could be refactored once we have govuk-components

Reply:
We've got it now!

Reply:
Possibly one to come back to in the next PR — we should apply consistently across the app?